docs/products/mysql/howto/create-tables-without-primary-keys.rst

```diff
@@ -0,0 +1,31 @@
+Create new tables without primary keys
+======================================
+
+Depending on when your Aiven for MySQL service has been created, it may by default not allow creating new tables without primary keys. 
```

Comment:
Is there a flag for this setting? Maybe defining it upfront would show people the flag they're looking for

Reply:
I will investigate this and get back to you

Reply:
I added now